nixos/modules/services/web-apps/dex.nix

```diff
@@ -31,9 +40,17 @@ in
       '';
     };
 
+    mount_postgresql = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Should we automatically mount /var/run/postgresql for a local postgresql server?
+      '';
+    };
```

Comment:
If we just ignore the error, we don't need this extra option.

Reply:
Yes, we can ignore the error in `BindPaths`, but I don't think we can ignore the error in `after`.

Reply:
hmmm, true.